{"uid":"23816bab924378ca","name":"test_valid_login","fullName":"tests.login_test.TestLogin#test_valid_login","historyId":"18657c207d70aa57845e0b7f4751c290","time":{"start":1753433335451,"stop":1753433335451,"duration":0},"description":"Test to verify that a user can log in with valid credentials.","descriptionHtml":"<p>Test to verify that a user can log in with valid credentials.</p>\n","status":"broken","statusMessage":"AttributeError: 'ConfigReader' object has no attribute 'get'","statusTrace":"self = <tests.login_test.TestLogin object at 0x10397e490>\nrequest = <SubRequest 'setup' for <Function test_valid_login>>\n\n    @pytest.fixture(autouse=True)\n    def setup(self, request):\n        config_reader = ConfigReader()\n        self.config = config_reader\n        self.driver = webdriver.Chrome()\n        self.driver.maximize_window()\n>       self.driver.get(self.config.get(\"base_url\"))\n                        ^^^^^^^^^^^^^^^\nE       AttributeError: 'ConfigReader' object has no attribute 'get'\n\nbase/base_test.py:12: AttributeError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"setup","time":{"start":1753433335452,"stop":1753433339994,"duration":4542},"status":"broken","statusMessage":"AttributeError: 'ConfigReader' object has no attribute 'get'\n","statusTrace":"  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pluggy/_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pluggy/_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/pluggy/_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/_pytest/fixtures.py\", line 1195, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/Library/Frameworks/Python.framework/Versions/3.13/lib/python3.13/site-packages/_pytest/fixtures.py\", line 922, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/Users/nguyenbahailong/Downloads/OutputTest/SauceDemo/base/base_test.py\", line 12, in setup\n    self.driver.get(self.config.get(\"base_url\"))\n                    ^^^^^^^^^^^^^^^\n","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"afterStages":[],"labels":[{"name":"story","value":"Valid Login"},{"name":"feature","value":"Login Tests"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"login_test"},{"name":"subSuite","value":"TestLogin"},{"name":"host","value":"MacBook-Pro-cua-Nguyen-2.local"},{"name":"thread","value":"64453-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.login_test"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":true,"retry":true,"extra":{"categories":[],"tags":[]},"source":"23816bab924378ca.json","parameterValues":[]}